### PR TITLE
Add FAQ entries for CEC vs Sandia module dbs and multi-inverter systems

### PR DESCRIPTION
**Pull request by kanderso-nrel**
- [ ] Closes #xxxx
 - [ ] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [ ] Tests added
 - [ ] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/main/docs/sphinx/source/reference) for API changes.
 - [ ] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [ ] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

What it says in the title.

### CODE REVIEW

**Review by cwhanse — APPROVED** [review]

File: docs/sphinx/source/user_guide/faq.rst [hunk 1]
```diff
@@ -138,3 +138,24 @@ so you can check the tables there for more up-to-date tables.
...
+
+
+Which should I use, the CEC or the Sandia PV Module database?
+-------------------------------------------------------------
+
+The CEC PV module database contains parameters for significantly more
+modules, and is more up to date, than the Sandia PV module database.
+Therefore, the CEC PV module database is probably the more useful option
+in most cases.  However, finding parameters for the specific module
+being used is more important than which database they came from.
```
Comment (mikofski) [inline docs/sphinx/source/user_guide/faq.rst:150]: Also, another distinction between the coefficients is that the CEC PV module database provides parameters for the [SAM/CEC 6-param model](https://www.nrel.gov/docs/fy15osti/64102.pdf) which yields an continuous IV curve whereas the Sandia PV module database provides parameters for the [SAPM](https://energy.sandia.gov/wp-content/gallery/uploads/SAND-2004_PV-Performance-Array-Model.pdf) which is a point model, yielding only max power point, open circuit, short circuit, and two optional intermediate points. Therefore, although both models are useful for finding the max power point, the SAM/CEC 6-param model could be more appropriate when exploration of off-MPP operating conditions is of interest, and the SAPM model could be more useful when just want max power is desired due to it's less computationally expensive yet sufficiently accurate algorithm.

### SAM/CEC 6-param model
image

### SAPM model
image

File: docs/sphinx/source/user_guide/faq.rst [hunk 2]
```diff
@@ -138,3 +138,24 @@ so you can check the tables there for more up-to-date tables.
...
+
+
+How do I model a system with multiple inverters?
+------------------------------------------------
+
+Currently, pvlib's :ref:`modelchaindoc` and :ref:`pvsystemdoc` only support
+simulating one inverter at a time.  To simulate a system with multiple
+inverters, define one ``PVSystem`` and ``ModelChain`` per inverter and
+run the simulation for each of them individually.  From there you
+can add up the inverter-level outputs to get the total system output.
```
Comment (wholmgren) [inline docs/sphinx/source/user_guide/faq.rst:160]:
```suggestion
can add up the inverter-level outputs to get the total system output.
If the modules, mounting, stringing, and inverters are all identical, then you
may simply multiply the ``ModelChainResult.ac`` by the number of inverters
to get the total system output.
```

Comment (wholmgren) [reply docs/sphinx/source/user_guide/faq.rst:160]: I suspect that most users are in this situation, so maybe reorder the two options? I don't have strong feelings about it.

**Review by wholmgren — APPROVED** [review]
Thanks @kanderso-nrel for the helpful contributions. I suggest that you merge this once you think it's ready.

**Pull request merged**